docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,121 @@
+
+![](./images/azure_client_secret.png)
+
+Once the Client Secret is created, the Client secrets section is displayed and you can find the Client Secret Value as shown below
+
+![](./images/azure_client_secret_value.png)
+
+Assign Access Control (IAM) role to the storage account. For more information, see [Assign Azure roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition)
+
+Go to the storage account that is created and click on Access control (IAM) as shown below
+
+![](./images/azure_iam_tab.png)
+
+Add role assignment and select the role `Storage Blob Data Contributor` as shown below
+
+![](./images/azure_add_role_assign.png)
+
+![](./images/azure_storage_role.png)
+
+On the Members tab, select User, group, or service principal to assign the selected role to the app registration. Click on Select members, and find your app registration name by typing in the Select box as shown below 
+
+![](./images/azure_member_assign.png)
+
+## Step 3: Set credentials
+
+Once that's completed, you'll need to set your Azure credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AZURE_STORAGE_TENANT_ID="your-app-tenant-id"
+export AZURE_STORAGE_CLIENT_ID="your-app-client-id"
+export AZURE_STORAGE_CLIENT_SECRET="your-app-client-secret-value"
+```
+
+For more information, see the official Azure documentation about [how to work with environmental credentials](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet).
+
+```{note}
+Kedro-Viz uses Gen2 filesystem protocol `abfs` to write files on AzureBlobStorage.
+```
+
+```{important}
+Having a `$web` container in your AzureBlobStorage is mandatory to use Kedro-Viz publish and share feature on Azure. For more information, see the official Azure documentation about 
+[Setting up a static website](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website#setting-up-a-static-website).
+```
+
+### Permissions and access control
+
+All permissions and access control are controlled by Azure. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
+
+You can control who can view your visualisation using [attribute-based access control](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-auth-abac). See the official Azure documentation for more information.
+
+### Billing
+
+You pay for storing objects on your AzureBlobStorage. The amount you pay depends on the volume of data stored per month, quantity and types of operations performed, along with any data transfer costs, data redundancy option selected.
+
+See the official [Azure documentation](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/) for more information.
+
```

Comment:
It would be nice to have `### Permissions and access control` and `### Billing` section before `## Step 3: Set credentials` and repeat the same for other cloud providers

Reply:
I've kept `### Permissions and access control` before `## Set credentials` and left `### Billing` were it is.